README.md

```diff
@@ -1 +1,64 @@
 # nextcloud-shibboleth-sync
+
+This repo contains an app to synchronize a list of identity providers with the 
+[user_saml](https://github.com/nextcloud/user_saml) Nextcloud application.
+
+This app is written in pure php without dependency and connect to a postgresql database.
+
+## How to build the image
+
+The docker image will copy the `app` directory and the `CMD` instruction directly executes the script.
+
+To build it, run:
+
+```bash
+$ docker build -t fundocker/nextcloud-shibboleth-sync:{TAG} .
```

Comment:
`TAG` is an example tag or an environment variable?

Reply:
Also: why not using `make build`?

Reply:
Because the `make build` build also the `nextcloud`. It is here only to drive the development task

Reply:
Maybe this Makefile target should be split into two sub-tasks? You did not answer my first question about what `{TAG}` means to you.

Reply:
sorry, `{TAG}` should be replaced with a real tag, I choose a template like syntax, maybe you have something better to suggest ?

Reply:
the thing is that having `{}` suggest variables to me, why not something like:

```suggestion
$ docker build -t fundocker/nextcloud-shibboleth-sync:my_tag .
```

Reply:
without `$` ? 

![](https://media.giphy.com/media/l1KsOGE23suZQeCfm/giphy.gif)

Reply:
I made the change.

Reply:
This was confusing to me :shrug: